### ISSUE TITLE
Experiment hyperparameters

### ISSUE THREAD

**Issue by danielvhvs** [git-disl/recap#6]
I had a question regarding the hyperparameters from the paper. Looking at the code there are some hyper parameters that I missed. For the experimental results in the paper, what are the hyperparameters you used for: The amount of iterations in init_pruning (num_pi) for the amount of total iterations over the full pruning finetuning loop (num_pr) for the amount of evaluation steps in the config (training_steps) the amount of training epochs per iteration in the loop in the config (num_train_epochs) and the batch size.

And what method do you use for looking at the memory size on gpu?
Do you use nvidia-smi or torch.cuda.max_memory_allocated()? Because there is no metric logger in the script.

There was a mention in the paper about an appendix. I got the paper from the ieee/cvf proceeding, which does not have an appendix, is there some alternative version that does have an appendix with the extra information about the experiments?

**Comment by danielvhvs** [comment]
I tried to run with some example parameters but when I compare that with the baseline (which I use prepruning using taylor pruning since I am not sure what baseline you used) it seems to perform worse so I am still curious what hyperparameters you used since I must be using wrong hyperparameters and baselines as I am not getting the same results.
